docs/source/precalculations/text_metadata.rst

```diff
@@ -0,0 +1,104 @@
+.. _text_metadata:
+
+Text Metadata Precalculation
+============================
+
+Description
+-----------
+
+This precalculation computes metadata about a text. 
+Since many checks cannot work with textual columns, this precalculation is used to numerize and categorize the texts.
+It generates new columns with data about the textual column, such as:
+
+NumCharsMetadata
+    Returns the total number of characters in the text.
+
+RatioUppercaseLettersMetadata
+    Returns the number of uppercase characters divided by the number of alphabetical characters.
+
+UnicodeCategoriesMetadata
+    Returns a string, containing the unicode categories of the characters in the text, comma seperated, sorted descending by their frequency.
+    What categories exist can be looked up here: https://www.fileformat.info/info/unicode/category/index.htm
+    Example:
+        input: 'Front-line leading edge website'
+        
+        output: 'Ll, Zs, Lu, Pd'
+
+UnicodeBlocksMetadata
+    Returns a string, containing the unicode blocks of the characters in the text, comma seperated, sorted descending by their frequency.
+    What blocks exist can be looked up here: https://www.fileformat.info/info/unicode/block/index.htm
+    
+    Example:
+        input: 'Front-line leading edge website'
+        
+        output: 'Basic Latin'
+
+NumWordsMetadata
+    Returns the total number of words in the text. A word is surrounded bz spaces or punctuation.
+    Hyphens count as word delimiters, apostrophes do not count as delimiters.
+
+DistinctWordsRatioMetadata
+    Returns the number of distinct words devided by the total number of words.
+
+UniqueWordsRatioMetadata
+    Returns the number of hapax legomenon, words that only appear once in the text, devided by the total number of words.
+
+UnknkownWordRatioMetadata
+    Returns the number of words that are not in a dictionary devided by the total number of words.
+    Popular reasons for words to be unknown: Typos, technical terms, neologisms.
+
+StopwordRatioMetadata
+    Returns the number of stopwords devided by the total number of words.
+    Stopwords are defined by the nltk stopwords, supporting multiple languages. 
```

Comment:
```suggestion
    Stopwords are defined by the nltk stopwords <word>, supporting multiple languages. 
```

is it a library? Then <word> should be 'library'...

you could also remove the 'the'